data/match/src/commonMain/graphql/EventSetsQuery.graphql

```diff
@@ -2,7 +2,7 @@ query EventSetsQuery(
     $eventId: ID
 ) {
     event(id: $eventId) {
-        sets {
+        sets(perPage: 100) {
```

Comment:
Instead of this really high number, if we don't want to build pagination, maybe we can make two requests, one which gets the total number of matches, and another to fetch all of those.